src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -1493,7 +1493,7 @@ private async ValueTask FlushThenWriteWithoutBufferingAsync(ReadOnlyMemory<byte>
         private ValueTask WriteHexInt32Async(int value, bool async)
         {
             // Try to format into our output buffer directly.
-            if (Utf8Formatter.TryFormat(value, _writeBuffer.AvailableSpan, out int bytesWritten, 'X'))
+            if (value.TryFormat(_writeBuffer.AvailableSpan, out int bytesWritten, 'X'))
```

Comment:
Curious why this build error is happening? Seems it's binding to a different `TryFormat` method?